### ISSUE TITLE
Floating point exception (core dumped)

### ISSUE THREAD

**Issue by wu-shuqi** [Yugnaynehc/ssta-captioning#9]
run :python train.py and get an error like this:
Learning rate: 0.0003
Batch size: 100
loading annotations into memory...
0:00:00.152458
creating index...
index created!
epoch:0 epsilon:0.96000000
Floating point exception (core dumped)
